### ISSUE TITLE
Context menu items are defined as buttons instead of menu items

### ISSUE THREAD

**Issue by jayspadie** [Microsoft/monaco-editor#446]
monaco-editor version: 0.8.3
Browser: Any
OS: Windows 10

During accessibility testing, we were told that Monaco's context menu should follow the menu pattern, including aria roles for menu and menuitems, instead of button roles.

This may be subjective, but the User Impact give was:
"When focus is on menu item narrator reads as "button".
When narrator says button, user cannot identify there is a second option in the context menu."

Shown below is what NVDA announces with Firefox.

![image](https://cloud.githubusercontent.com/assets/5246869/26266574/e82f6446-3cb3-11e7-91eb-1e35c58c0e45.png)
